examples/public/stylesheets/main.less

```diff
@@ -49,3 +50,16 @@ body,
   height: 100%;
   font-family: @global-monospaced-font-face;
 }
+
+.toolbar {
+  position: absolute;
+  @top-offset: 160px;
+
+  top: @top-offset;
+  @media (max-height: (@top-offset / 0.22)) {
+    top: 22%;
+  }
+  right: 40px;
+
+  max-height: ~"calc(100% - "(@top-offset - 40px)~")";
```

Comment:
This doesn't actually work. If anyone can help. I'm trying to have the height of panel from where the panels start at 160px from the top down to 40px from the bottom. It does't work because all the contents of the toolbar are absolute positioned.